.github/CODEOWNERS

```diff
@@ -43,6 +43,9 @@ xptifw/ @intel/llvm-reviewers-runtime
 
 # DPC++ tools
 llvm/ @intel/dpcpp-tools-reviewers
+llvm/include/llvm/SYCLLowerIR/MutatePrintfAddrspace.h @intel/dpcpp-tools-reviewers
+llvm/lib/SYCLLowerIR/MutatePrintfAddrspace.cpp @intel/dpcpp-tools-reviewers
+llvm/test/SYCLLowerIR/printf_addrspace/ @intel/dpcpp-tools-reviewers
```

Comment:
Shouldn't we just remove 

SYCLLowerIR/ @intel/dpcpp-esimd-reviewers

?

Reply:
It would look simpler, however wouldn't this break ESIMD code-ownership rules in turn?

Reply:
I have no idea, so I suggest to test it.
It looks like we have a mix of passes in SYCLLowerIR directory and it would be great if we follow some naming pattern allowing us match ESIMD passes (e.g. put ESIMD specific passes to ESIMD directory or each passes name should start with or contain ESIMD, etc.).
In my opinion, adding a new rule for each new file introduces overhead on reviewers and code authors.

Reply:
+1 on moving ESIMD-specific passes into a separate subdirectory (either within `SYCLLowerIR` or alongside with it)

Reply:
I've liked the approach with ESIMD-specific directories - please check if the new implementation is better. Also encouraging other code-owner teams to review, especially the ESIMD team :)